packages/transformers/js/src/JSTransformer.js

```diff
@@ -248,6 +252,8 @@ export default (new Transformer({
     let rootPkg = result?.contents;
 
     let inlineEnvironment = config.isSource;
+    const relativeContext =
```

Comment:
I couldn't get this to work with `rootPkg` (like a couple of lines below)

```
relativeContext = rootPkg['@parcel/transformer-js']?.relativeContext ?? relativeContext;
```

This didn't work :/